doc/lammps.1

```diff
@@ -226,6 +226,20 @@ arguments of the "dump" command. See the
 .B LAMMPS
 manual for details on either of the two commands.
 .TP
+\fB\-r2info <restart file> <keyword> ...\fR or
+\fB\-restart2info <restart file> <keyword> ...\fR
+Write infomation about the <restart file> previously written by
```

Comment:
typo: information

Reply:
@jtclemm I am amazed by your ability to spot these kinds of typos. I noticed I make them regularly and am debating whether it is my "fast reading of words by shape rather than by letter" that makes me miss them or that the now almost 30 year old [IBM Model-M keyboard](https://en.wikipedia.org/wiki/Model_M_keyboard) that I am typing this on is finally starting to wear out. But then again, the one in the office is five years older and I have been using it since I was a graduate student .

Reply:
Unfortunately any ability I may have to spot typos does not translate to not making typos myself ;)